Caddyfile

```diff
@@ -0,0 +1,71 @@
+{
+    order cache before rewrite
```

Comment:
Maybe that using an ordered section (writhing brackets) would make the Caddyfile simpler?